tests/unit/test_materialized_view.py

```diff
@@ -53,3 +54,52 @@ def test_redshift_materialized_view_config_throws_expected_exception_with_invali
     )
     with pytest.raises(ValueError):
         config.parse_relation_config(model_node)
+
+
+def test_redshift_materialized_view_parse_relation_results_handles_multiples_sort_key():
+    materialized_view = agate.Table.from_object(
+        [],
+        [
+            "database",
+            "schema",
+            "table",
+            "diststyle",
+            "sortkey1",
+            "autorefresh",
+        ],
+    )
+
+    column_descriptor = agate.Table.from_object(
```

Comment:
In light of the `.get()` comment above, let's add a third column here that is not part of the sort key. I tend to forget negative testing and that's what winds up biting us in the end.